src/main/java/org/folio/ebsconet/mapper/OrdersMapper.java

```diff
@@ -103,8 +104,8 @@ public void ebsconetToFolio(MappingDataHolder mappingDataHolder) {
         .fundId(fund.getId())
         .code(fund.getCode())
         .expenseClassId(mappingDataHolder.getExpenseClass())
-        .distributionType(FundDistribution.DistributionTypeEnum.AMOUNT)
-        .value(ebsconetOrderLine.getUnitPrice().multiply(BigDecimal.valueOf(ebsconetOrderLine.getQuantity())));
+        .distributionType(FundDistribution.DistributionTypeEnum.PERCENTAGE)
+        .value(ONE_HUNDRED_PERCENT);
```

Comment:
Here is code that works with fund distribution when creating ebsconet order from common purchase order
![image](https://user-images.githubusercontent.com/25097693/186198276-c3ca653e-bc1e-477c-8eb8-f8277cc20875.png)
So only fund code necessary for ebsconet and so there should not be any compatibility issues